qa/kerberos/src/itest/java/org/elasticsearch/hadoop/qa/kerberos/AbstractClusterVerificationTests.java

```diff
@@ -69,14 +69,16 @@ public void testOutputContents() throws Exception {
                     successMarker.exists());
         }
 
-        File dataFile = new File(integrationDataDir, dataFileName);
-        Assert.assertTrue("Expected data file [" + dataFile + "] to exist but could not find it.", dataFile.exists());
-        BufferedReader reader = new BufferedReader(new InputStreamReader(new FileInputStream(dataFile)));
         int lines = 0;
-        while (reader.readLine() != null) {
-            lines++;
+        for (File directoryFile : integrationDataDir.listFiles()) {
```

Comment:
Might be easier to do something like

```java
final LongAdder lines = new LongAdder();
Files.walk(Paths.get("foo"))
    .filter(p -> p.startsWith(prefix))
    .forEach(p -> {
        try (Stream<String> line = Files.lines(p)) {
            line.forEach(l -> lines.increment());
        } catch (IOException e) {
            fail("got: " + e);
        }
    });
```

(totally up to you though)

Reply:
I left it simplified without lambdas to make the backport to the 6.7 branch easier, it's still on java 6. ☹️